grid-cli/cmd/deploy_kubernetes.go

```diff
@@ -186,33 +185,30 @@ var deployKubernetesCmd = &cobra.Command{
 			masterNode = uint32(nodes[0].NodeID)
 		}
 		master.Node = masterNode
-		if workersNode == 0 && len(workers) > 0 {
-
+		if len(workersNodes) < workerNumber {
```

Comment:
&& workersNumber > 0, no need to filter if no nodes are requested in the first place no?

Reply:
okay but (len(workersNodes) < workerNumber) satsify this as
if workersnumber <= 0 -> no need to filter->  len(workersNodes) will be 0 or greater and condition will be false
if workersnumber > 0 -> it will go only if no workersNodes is less than no workers which is right